test/dbscan.jl

```diff
@@ -46,3 +46,9 @@ clusters = dbscan(points, 0.3, min_neighbors=2, min_cluster_size=100, leafsize=2
 @test clusters[1].core_indices == inds_1
 @test clusters[2].core_indices == inds_2
 @test clusters[3].core_indices == inds_3
+
+# Issue #84
+clusters = dbscan(convert(Matrix{Float32}, points), 0.3f0, min_neighbors=2, min_cluster_size=100, leafsize=20)

```

Comment:
Since we are allowing any radius, it would be nice to test that `Matrix{Float32}` / `Float64` combination also works (the actual error in #84).

Reply:
I added one.